references/optical_flow/transforms.py

```diff
@@ -109,7 +114,8 @@ class RandomErasing(T.RandomErasing):
     def __init__(self, p=0.5, scale=(0.02, 0.33), ratio=(0.3, 3.3), value=0, inplace=False, max_erase=1):
         super().__init__(p=p, scale=scale, ratio=ratio, value=value, inplace=inplace)
         self.max_erase = max_erase
-        assert self.max_erase > 0
+        if not self.max_erase > 0:
```

Comment:
I guess normally we would write it this way?

```suggestion
        if self.max_erase <= 0:
```